frontend/tests/unit/services/transactions/request-manager.spec.ts

```diff
@@ -42,6 +42,14 @@ const PROVIDER = new JsonRpcProvider();
 const SIGNER = new JsonRpcSigner(undefined, PROVIDER);
 const DEFAULT_REQUEST_IDENTIFIER = '1';
 
+function mockGetSafeEventHandler() {
```

Comment:
Can't we put such functions to a shared utility file, so we don't have to define them several times?